docs/subtensor-nodes/using-docker.md

```diff
@@ -16,27 +16,27 @@ If you are already running a subtensor node using Docker, then go directly to [S
 
 ## Step 1: Install git
 
-Make sure you installed `git` on your machine. See [GitHub docs](https://docs.github.com/en/get-started).
+Ensure that git is installed on your machine. Refer to the [GitHub documentation](https://docs.github.com/en/get-started) for installation instructions..
```

Comment:
I usually put "git" in code ticks only because we are not using "git" in English word sense. So I recommend we use `git` here. Rest looks great, thanks!

Reply:
Thank you. Fixed.